app/oxide-rot-1/app.toml

```diff
@@ -68,6 +69,7 @@ task-slots = ["syscon_driver"]
 [tasks.sprot]
 name = "drv-lpc55-sprot-server"
 priority = 6
+max-sizes = {flash = 54016, ram = 32768}
```

Comment:
Do we really need to reintroduce these or can we rely on autosizing?

Reply:
They don't need to be here. Deleted.